sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -48,6 +55,40 @@ def _from_generated(cls, generated):
         )
 
 
+class LogsTableRow(object):
+    """Represents a single row in logs table.
+
+    ivar list row: The collection of values in the row.
+    ivar int row_index: The index of the row in the table
+    """
+    def __init__(self, **kwargs):
+        # type: (Any) -> None
+        _col_types = kwargs.pop('col_types', None)
+        row = kwargs.get('row', None)
+        self.row = process_row(_col_types, row)
+        self.row_index = kwargs.get('row_index', None)
```

Comment:
Is it by design that row_index is null-able?

Reply:
none of these can be none - changed all of them